.github/workflows/jruby.yml

```
@@ -0,0 +1,29 @@
+name: build
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '12 3 * * *'
+
+jobs:
+  build:
+    name: build (${{ matrix.ruby }} / ${{ matrix.os }})
+    strategy:
+      matrix:
+        ruby: [ 'jruby-9.3' ]
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1.82.0
```

```suggestion
      uses: ruby/setup-ruby@v1
```
and it's also what's used in the other workflow.
(https://github.com/ruby/setup-ruby/#versioning for details)